.github/workflows/stale.yml

```diff
@@ -16,7 +16,11 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'Stale issue message'
-        stale-pr-message: 'Stale pull request message'
-        stale-issue-label: 'no-issue-activity'
-        stale-pr-label: 'no-pr-activity'
+        stale-issue-message: 'This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.'
+        stale-issue-label: 'stale issue'
+        stale-pr-label: 'stale pull request'
+        exempt-issue-labels: 'ready to merge,read for copy-edit,paused,in progress,linked pull request'
+        exempt-pr-labels: 'ready to merge,read for copy-edit,paused,in progress'
```

Comment:
I guess these should have said "ready for ..." rather than "read for ..." ?

Reply:
Fixed in 33ffe2b